src/Compilers/CSharp/Portable/Compilation/CSharpCompilerDiagnosticAnalyzer.cs

```diff
@@ -36,7 +36,15 @@ internal override ImmutableArray<int> GetSupportedErrorCodes()
                         // We don't support configuring WRN_ALinkWarn. See comments in method "CSharpDiagnosticFilter.Filter" for more details.
                         continue;
 
+                    case (int)ErrorCode.WRN_UnreferencedField:
```

Comment:
Because this one isn't exposed in the CommonMessageProvider base class. Perhaps an additional member `IsNotSupportedByLiveAnalysis` to that abstract class would be better with an overload in the C# one? There doesn't appear to be any specific to VB.